### ISSUE TITLE
Loss function calculation still wrong

### ISSUE THREAD

**Issue by farhantandia** [ktjonsson/keras-ArcFace#3]
I've test your defined lost function, i got 100% on validation but 0.00 on testing using another metric, please check again.
and also there are one repo that shows correct arc loss function in here https://github.com/4uiiurz1/keras-arcface.
How come you can get almost 100% only 1-3 iteration not only on Cifar10 , I also testing using large dataset such Market1501 (metric = mAP).
Thanks.
